packages/asset/contracts/Asset.sol

```diff
@@ -350,4 +351,19 @@ contract Asset is
     function isBridged(uint256 tokenId) external pure returns (bool) {
         return TokenIdUtils.isBridged(tokenId);
     }
+
+    /// @notice This function is used to register Asset contract on the Operator Filterer Registry of Opensea.can only be called by admin.
+    /// @dev used to register contract and subscribe to the subscriptionOrRegistrantToCopy's black list.
+    /// @param subscriptionOrRegistrantToCopy registration address of the list to subscribe.
+    /// @param subscribe bool to signify subscription "true"" or to copy the list "false".
+    function registerAndSubscribe(address subscriptionOrRegistrantToCopy, bool subscribe) external onlyRole(DEFAULT_ADMIN_ROLE) {
+        require(subscriptionOrRegistrantToCopy != address(0), "Asset: subscription can't be zero address");
+        _registerAndSubscribe(subscriptionOrRegistrantToCopy, subscribe);
+    }
+
+    /// @notice sets filter registry address deployed in test
+    /// @param registry the address of the registry
+    function setOperatorRegistry(address registry) external onlyRole(DEFAULT_ADMIN_ROLE) {
+        operatorFilterRegistry = IOperatorFilterRegistry(registry);
```

Comment:
Need to add validation

Reply:
And tests :)

Reply:
Misisng: Only default admin should be able to set role / if not default admin should not be able to set role